scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -382,12 +382,22 @@
             :to-make-room-for pending-job)]
     [state nil]))
 
+(defn reserve-hosts! [state preemption-decisions]
+  (let [multiple-tasks (filter #(< 1 (count (:task %))) preemption-decisions)
```

Comment:
Nitpick: rename to `multiple-task-decisions`

Reply:
Done